utils/gridlink_utils.h.src

```diff
@@ -20,9 +20,10 @@ extern "C" {
 #include "weight_defs_DOUBLE.h" //for definition of weight_struct
 
     extern int get_binsize_DOUBLE(const DOUBLE xdiff,
+                                  const DOUBLE xwrap,
                                   const DOUBLE rmax,
-                                  const int refine_factor, const int max_ncells, DOUBLE *xbinsize, int *nlattice,
-                                  const struct config_options *options) __attribute__((warn_unused_result));
```

Comment:
What was `options` being used for? Otherwise, it should have raised an unused parameter warning before